### DISCUSSION TITLE
Detection accuracy?

### DISCUSSION THREAD

**Discussion by gosssamer** [mcguirepr89/BirdNET-Pi#1021] [General]
Hi, I'm a new birdnet user on my raspberry pi and have played a few sample sounds from youtube and elsewhere to test and make sure it's working, but it failed to properly detect all two sample bird sounds I played.

I tried to play sounds from local birds here in NJ, including a bluejay and cardinal, but it detected a cardinal as a "Madagascar nightjar" and a bluejay as a "Victorin's warbler". I also made sure there was a sufficiently long sample to be sure.

Any idea how to explain this? I was thinking perhaps these were the genus or species names for the birds, but that doesn't appear to be the case.

Here is the recording of bluejays I used. I think it was recorded as quite a few different types of birds, even though they were clearly all bluejays.
https://youtu.be/F-ydPOkoW1o?si=CKxXrEG9wBZI6KlQ

Perhaps there's a known recording that you use as a test that I could try to make sure?

**Comment by gosssamer** [comment]
Hi - does anyone have any input on the overall accuracy of birdnet and why random sounds might be interpreted as birds that don't even exist in my region? How do you "tune" the system?

**Reply by ehpersonal38** [reply]
This should never really happen, especially for the most common birds.

Try the steps here and see if it helps:
https://github.com/mcguirepr89/BirdNET-Pi/discussions/1028#discussioncomment-7085420
